packages/admin/package.json

```diff
@@ -22,6 +22,7 @@
     "react": "^17.0.1",
     "react-admin": "^3.10.4",
     "react-dom": "^17.0.1",
+    "react-final-form": "^6.5.3",
```

Comment:
There's no other way to retrieve form values using react-admin? Any exposed API?

Reply:
Nice, @luizeboli! 

Make sense. I refactored this piece, making the form values retrieving without react-final-form dep.

Now, I'm using the FormDataConsumer from RA. 🤟🏻